mscore/keyb.cpp

```diff
@@ -214,6 +214,17 @@ void ScoreView::editKey(QKeyEvent* ev)
                   }
             }
 
+
+#ifdef Q_OS_WIN // Japenese IME on Windows uses special keys, so musescore should ignore
+      if (editObject->isText() && mscore->getInputMethodScript() == QLocale::JapaneseScript &&
+          ((key == Qt::Key_Control || (modifiers & Qt::ControlModifier)) ||
+           (key == Qt::Key_Alt || (modifiers & Qt::AltModifier)) ||
+           (key == Qt::Key_Shift || (modifiers & Qt::ShiftModifier)) ||
+           (key == Qt::Key_CapsLock))) {
+            return;
+            }
+#endif
```

Comment:
I'm having TextEdit ignore Contrl, Alt, Shift, and CapsLock.  I couldn't find definitive microsoft documentation only of all keys that Windows' IME uses, though.  If we new the exact specific keys, then could just do those.  Also, I'm wondering if there are other langauges that Windows' IME needs special keys...maybe would want to include those here too.